.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
```

Comment:
The Gitter chat is mentioned in the README. I think we're primarily trying to steer people to Discourse for questions as its more visible. If someone asks a question in Gitter, it will get lost in the chat history, whereas on Discourse it would be quite easy to discover even after several years.